pyproject.toml

```diff
@@ -8,7 +8,7 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "ops-scenario"
 
-version = "7.0.1"
+version = "7.0.2"
```

Comment:
Should this be 7.1, as we're adding things like `ExecArgs`?

Reply:
I guess it could be. I don't see that as adding functionality - it's just vendoring a class. If you prefer 7.1 then I'm fine with that.

Reply:
Yep, fair enough.